docs/examples/prototypes.md

```diff
@@ -93,7 +93,6 @@ We have many other prototypes or demo apps that developers in our community buil
 | Flight Seatmap display                     | SeatMap Display                                                                                   | React                | [amadeus-seatmap](#amadeus-seatmap)                                                                                                  |
```

Comment:
Let's update also the official prototypes list? In the table of the official prototypes as well as the details later that we explain what the prototype does + the gif. You can find the new gif + new name of the prototype here: https://github.com/amadeus4dev/amadeus-hotel-search-pois-tours-django/pull/35